lib/addoninfo.cpp

```diff
@@ -27,19 +27,29 @@
 
 #include "json.h"
 
-static std::string getFullPath(const std::string &fileName, const std::string &exename) {
+static std::string getFullPath(const std::string &fileName, const std::string &exename, bool debug = false) {
+    if (debug)
+        std::cout << "looking for addon '" << fileName << "'" << std::endl;
     if (Path::isFile(fileName))
         return fileName;
 
     const std::string exepath = Path::getPathFromFilename(exename);
+    if (debug)
+        std::cout << "looking for addon '" << (exepath + fileName) << "'" << std::endl;
```

Comment:
I Am curious why do you use + temporary instead of <<

Reply:
Bad copy & paste. I have not changed it since it is also done outside of this code so will check the whole code for this. I filed https://trac.cppcheck.net/ticket/12846 about detecting this.